doc_generator/mechanical/components/cadquery/nimble_assembly.py

```diff
@@ -27,10 +29,10 @@
 # Build the assembly
 assy = cq.Assembly()
 assy.add(bottom, name="bottom_end")
-assy.add(create_beam(total_height), name="leg1", loc=cq.Location((-width / 2.0 + 10, -depth / 2.0 + 10, 3)))
-assy.add(create_beam(total_height), name="leg2", loc=cq.Location((width / 2.0 - 10, -depth / 2.0 + 10, 3)))
-assy.add(create_beam(total_height), name="leg3", loc=cq.Location((width / 2.0 - 10, depth / 2.0 - 10, 3)))
-assy.add(create_beam(total_height), name="leg4", loc=cq.Location((-width / 2.0 + 10, depth / 2.0 - 10, 3)))
+assy.add(make_rack_leg(total_height), name="leg1", loc=cq.Location((-width / 2.0 + 10, -depth / 2.0 + 10, 3)).cq())
```

Comment:
It looks like Assembly was removed in the release version of CadScript. Will that be added back later so that these can be converted to CadScript Assembly calls?

Reply:
It is still in, but it is still super simplistic. Will add documentation and more functionality later